src/config/env.js

```
@@ -0,0 +1,16 @@
+const port = process.env.PORT || 8080;
+
+const defaultAppMsg = {
+  msg: process.env.DEFAULT_APP_MSG ||
+    `${process.env.NODE_ENV ? process.env.NODE_ENV : 'DEV'}: Circles user profile service`
+}
+
+const cognitoPoolId = process.env.COGNITO_POOL_ID;
```

maybe throw if these are not set?

---

That's a good idea, will add